integrations/langflow/pyproject.toml

```diff
@@ -52,6 +52,13 @@ dependencies = [
     "nest-asyncio>=1.6.0",
 ]
 
+[project.optional-dependencies]
+http = [
```

Comment:
These may not be necessary since Stepflow SDK has them? Or, maybe we need an optional dependency that enables an optional dependency of the stepflow SDK?